.doc_gen/metadata/s3_metadata.yaml

```diff
@@ -627,8 +824,40 @@ s3_PutBucketWebsite:
             - description:
               snippet_tags:
                 - s3.java2.set_website_configuration.main
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/s3
+          sdkguide: sdk-for-javascript/v3/developer-guide/s3-example-static-web-host.html#s3-example-static-web-host-set-website
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - s3.JavaScript.buckets.createclientv3
+            - description: Set the website configuration.
+              snippet_tags:
+                - s3.JavaScript.website.putBucketWebsiteV3
   services:
     s3: {PutBucketWebsite}
+s3_GetBucketWebsite:
+  title: Retrieve the website configuration for an &S3; bucket using an &AWS; SDK
+  title_abbrev: Get the website configuration for a bucket
+  synopsis: get the website configuration for an &S3; bucket.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/s3
+          sdkguide:
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - s3.JavaScript.buckets.createclientv3
+            - description: Get the website configuration.
+              snippet_tags:
+                - s3.JavaScript.website.getBucketWebsiteV3

```

Comment:
This snippet has an extra '*/' at the top, which is making the whole snippet render as a comment in the doc build.